physionet-django/notification/templates/notification/email/mailto_contact_supervisor.html

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}{% autoescape off %}
+Dear Prof. {{ application.reference_name }},
+
+{{ applicant_name }} is applying for credentialed access to PhysioNet, the website for sharing biomedical data. He/She have listed you as their supervisor and he/she have summarized the research project as follows:
```

Comment:
"He/She has"

Reply:
This is a exact copy paste for what I was given.

Reply:
Fair enough 👍